src/api/controllers/commentController.js

```diff
@@ -0,0 +1,185 @@
+import _ from 'lodash';
+import Sequelize from 'sequelize';
+import status from '../../helpers/constants/status.codes';
+import models from '../models';
+import sendError from '../../helpers/error.sender';
+import errorMessages from '../../helpers/constants/error.messages';
+import generatePaginationDetails from '../../helpers/generate.pagination.details';
+
+const { Comment, User } = models;
+
+/**
+ *
+ *
+ * @class CommentController
+ */
+class CommentController {
+  /**
+   * A controller to post a new comment on an article
```

Comment:
isn't better if you find something that includes all operations that are in this class, The description seems to me like it is just posted a comment. try something like ``` A controller to CRUD  comment on an article```

Reply:
Cool @murediane 
Let me do that